ambari-logsearch-docs/src/main/resources/templates/shipper_configurations.md.ftl

```diff
@@ -38,8 +136,97 @@ limitations under the License.
     </#list>
 </#if>
 
-### Filter Descriptor Sections
+### Filter Descriptor
 
+Filter configurations can be defined in the filter descriptor section.
+Example:
+```json
+{
+  "input" : [
+  ],
+  "filter": [
+    {
+      "filter": "json",
+      "conditions": {
+        "fields": {
+          "type": [
+            "simple_service_json"
+          ]
+        }
+      }
+    }
+    {
+      "filter": "grok",
+      "deep_extract": "false",
+      "conditions":{
+        "fields":{
+          "type":[
+            "simple_service",
+            "simple_audit_service",
+            "docker_service"
+          ]
+        }
+      },
+      "log4j_format":"%d{ISO8601} %5p [%t] %c{1}:%L - %m%n",
+      "multiline_pattern":"^(%{TIMESTAMP_ISO8601:logtime})",
+      "message_pattern":"(?m)^%{TIMESTAMP_ISO8601:logtime}%{SPACE}%{LOGLEVEL:level}%{SPACE}\\[%{DATA:thread_name}\\]%{SPACE}%{JAVACLASS:logger_name}:%{INT:line_number}%{SPACE}-%{SPACE}%{GREEDYDATA:log_message}",
```

Comment:
Including some lines of sample log can be useful to understand what log entries can be parsed with this pattern.